.github/workflows/alert-sync-issues.yml

```
@@ -1,14 +1,13 @@
+  workflow_dispatch: null
 permissions:
   contents: read
-
 jobs:
   sync:
     uses: KooshaPari/phenoShared/.github/workflows/reusable/alert-sync-issues.yml@6a6e1b06026e9443449e419a2892cd0e47c19442
     with:
-      auto-label: auto-alert-sync
\ No newline at end of file
+      auto-label: auto-alert-sync
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "timeout-minutes" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "sync"

(syntax-check)